exdb/tests.py

```diff
@@ -249,12 +249,58 @@ def create_sub_type(self):
     def create_org(self):
         return Organization.objects.create(name="Test Organization")
 
+    def create_keyword(self):
+        return Keyword.objects.create(name="Test Keyword")
```

Comment:
I assume that the point of this function and the ones like it is for reuse. Each one of these functions should use a get_or_create so that there can be consistency in things such as ids.